### PR TITLE
adding bytesize variantcatalogue

### PR DESCRIPTION
**Pull request by FranBonath**
@scorreard I did a bit of guesses in the description, so have a close look at that :)

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1571"></a>

### CODE REVIEW

File: markdown/events/2023/bytesize_variantcatalog.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+Just **15 minutes** + questions, we will be focussing on topics about using and developing nf-core pipelines.
+These will be recorded and made available at <https://nf-co.re>
+It is our hope that these talks / videos will build an archive of training material that can complement our documentation. Got an idea for a talk? Let us know on the [`#bytesize`](https://nfcore.slack.com/channels/bytesize) Slack channel!
+
+## Bytesize: variantcatalogue
+
+This week, Solenne Correard ([@scorreard](https://github.com/scorreard)) is introducing us to the nextflow pipeline `variantcatalogue`. Though not an nf-core pipeline (yet), `variantcatalogue` is using nf-core derived material to aid in the creation of the pipeline.
+
+The variant catalogue pipeline is a Nextflow workflow designed to generate variant catalogues, a list of variants and their frequencies in a population, from whole genome sequences. More information on the pipeline can be found [here](https://github.com/wassermanlab/Variant_catalogue_pipeline).
```
Comment (scorreard) [inline markdown/events/2023/bytesize_variantcatalog.md:25]:
```suggestion
The variant catalogue pipeline is a Nextflow workflow designed to generate variant catalogues (also called variant libraries): a list of variants and their frequencies in a population, from whole genome sequences. More information on the pipeline can be found [on Github](https://github.com/wassermanlab/Variant_catalogue_pipeline) or in the [associated preprint](https://www.biorxiv.org/content/10.1101/2022.10.03.508010v2).
```

**Review by scorreard — COMMENTED** [review]
Should it mention somewhere that it will not be live (because I am in Canada) but that I will be happy to answer questions on slack?

**Comment by scorreard** [comment]
Just suggested a small change to include the preprint that really details the pipeline. But I'm ok to not include it neither, you can decide what is your preference.

**Comment by FranBonath** [comment]
You are right, I will add that it will be recorded!

**Review by maxulysse — APPROVED** [review]
<3

**Pull request merged**